Makefile

```diff
@@ -359,7 +360,7 @@ ifeq ($(OS),win)
 
 		LICENSE_IN_USE := $(filter-out freeglut,$(LICENSE_IN_USE))
 	else
-		CXXLIBS += -mwindows -lopengl32 -lglu32 -lwinmm
+		CXXLIBS += -mwindows -lopengl32 -lglu32 -lwinmm -lcomdlg32 -lole32
```

Comment:
I think these guys are unnecessary :)

Reply:
Yeah. I noticed that too late. I already made the changes but was waiting for the checks to finish.